### PR TITLE
Update quickstart-install-connect-docker.md

### PR DESCRIPTION
**Pull request by teboho**
Container registry provides `images` which are then used to create `containers`

### CODE REVIEW

**Comment by ttorble** [comment]
@amitkh-msft 

Can you review the proposed changes?  

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge.  

#label:"aq-pr-triaged" 
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
Thanks for your contribution!

File: docs/linux/quickstart-install-connect-docker.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ For the bash commands in this article, `sudo` is used. If you don't want to use
 
 ## [CLI](#tab/cli)
 
-### Pull the container from the registry
+### Pull the image from the registry
```
Comment (rwestMSFT) [inline docs/linux/quickstart-install-connect-docker.md]:
```suggestion
### Pull the container image from the registry
```

File: docs/linux/quickstart-install-connect-docker.md [hunk 2]
```diff
@@ -593,7 +593,7 @@ For the bash commands in this article, `sudo` is used. If you don't want to use
...
```
Comment (rwestMSFT) [inline docs/linux/quickstart-install-connect-docker.md]:
```suggestion
### Pull the container image from the registry
```

**Pull request merged**